src/extract/src/html/extract-regexps-html.py

```diff
@@ -20,5 +22,12 @@
 with open('./src/html/temp-js-content.js', 'w') as fp:
     fp.write(js_from_html)
 
-# return the path to the temp-js-content.js based on the location of extract-regexes.pl
-print('./src/html/temp-js-content.js', end = '')
\ No newline at end of file
+# hardcoded js extractor location
+output = subprocess.run(['./src/javascript/extract-regexps.js', './src/html/temp-js-content.js'], capture_output=True, text=True)
```

Comment:
The only concern that I still have is that the location of the js regex extractor is hardcoded in the html regex extractor. I thought about letting the meta-program pass the location to the html regex extractor, but that would require creating a special case for html file in the meta-program, which I think would create the modular design.